### PR TITLE
parsedmarc: init at 6.10.0

### PR DESCRIPTION
**Pull request by fadenb**
###### Motivation for this change
Parsedmarc is not yet available in nixpkgs.
I plan to provide a service module soon.

###### Things done
* Added missing python dependencies
* Added parsedmarc

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/expiringdict/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  pname = "expiringdict";
+  version = "1.2.1";
+  disabled = !python.isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0l421pczpglfwqhl1kcb7wrfxg4554p6v7yjlg4c698448ks8azy";
+  };
+
+  doCheck = false;
```
Comment (jonringer) [inline pkgs/development/python-modules/expiringdict/default.nix:13]: There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.

File: pkgs/tools/misc/parsedmarc/default.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{ lib, buildPythonApplication, fetchPypi, mail-parser, expiringdict, xmltodict
+, lxml, mailsuite, dateparser, geoip2, publicsuffix2, elasticsearch-dsl
+, kafka-python, tqdm }:
```
Comment (jonringer) [inline pkgs/tools/misc/parsedmarc/default.nix:3]:
```suggestion
, kafka-python, tqdm }:

```

**Review by jonringer — COMMENTED** [review]
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

**Comment by fadenb** [comment]
There is some activity on #96533. Basically the lack of a new enough elastic held me back to push forward on this. I intend to pick it up again soon-ish as I have a renewed need for parsedmarc.

**Comment by fadenb** [comment]
Made obsolete by #125404 :)

**Pull request closed without merging**